logfire/__init__.py

```diff
@@ -21,6 +21,7 @@
 from ._internal.main import Logfire, LogfireSpan
 from ._internal.scrubbing import ScrubbingOptions, ScrubMatch
 from ._internal.utils import suppress_instrumentation
+from .current_span import current_logfire_span, current_span
```

Comment:
I think it could be confusing or cause weird side effects that the imported `current_span` overrides the `current_span` module.